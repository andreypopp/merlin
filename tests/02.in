["find", "use", "sexplib"]
["tell", "struct", "module Test_sexp = struct "]
["tell", "struct", "type t = int with sexp "]
["tell", "struct", "let test = sexp_of_t "]
["tell", "struct", "end"]
["tell", "struct", null]
["errors"]
