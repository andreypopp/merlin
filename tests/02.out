["return",true]
["return",false]
["return",false]
["return",false]
["return",false]
["return",true]
["return",[]]
